packages/suite/src/components/wallet/WalletLayout/AccountBanners/AccountBanners.tsx

```diff
@@ -25,18 +27,25 @@ type AccountBannersProps = {
     account?: Account;
 };
 
-export const AccountBanners = ({ account }: AccountBannersProps) => (
-    <BannersWrapper>
-        <CoinjoinContextMessage account={account} />
-        <AuthConfirmFailed />
-        <BackendDisconnected />
-        <DeviceUnavailable />
-        <TorDisconnected />
-        <XRPReserve account={account} />
-        <AccountImported account={account} />
-        <AccountOutOfSync account={account} />
-        <EvmExplanationBanner account={account} />
-        <TaprootBanner account={account} />
-        {account?.symbol && <StakeEthBanner account={account} />}
-    </BannersWrapper>
-);
+export const AccountBanners = ({ account }: AccountBannersProps) => {
+    const { pathname } = useSelector(state => state.router);
+
+    return (
+        <BannersWrapper>
+            {account?.accountType === 'coinjoin' && <ContextMessage context={Context.coinjoin} />}
+            {account?.symbol &&
+                ['eth', 'thol'].includes(account.symbol) &&
```

Comment:
Why not to use  existing `isSupportedNetworkSymbol` util?

Reply:
ff0c43c2e473a9dfa62db0feac39e1eb4a623ec7